jobs/deep-sea-fishing.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Deep Sea Fishing
+---
+
+| Întrebare   | Răspuns |
+| ----------- | :-----------: |
+| Câștig mediu pe ora fără bonusuri? | <Dinero :amount='1150' /> ~ <Dinero :amount='1440' /> |
+| Ce nivel este necesar? | Nivel 15 |
+| Necesită vehicul personal? | Da, barcă personală cu capacitate de depozitare. |
+| Necesită permis de conducere? | Nu. |
+| Necesită iteme specifice? | Nu. |
+| Posibilitate câștigare iteme extra? | Nu. |
+| Locație NPC Deep Sea Fishing | Port Los Santos |
+
```

Comment:
**Care este venitul din cadrul jobului „Deep Sea Fishing” in functie de skill-ul detinut?**

	Venitul jobului pasiv "Deep Sea Fishing", in functie de cresterea in skill, este la fel ca la joburile celelalte: Silver -> venit majorat cu 7%; Gold -> venit majorat cu 23%; Platinum -> venit majorat cu 53%; Diamond -> venit dublu; Master -> venit triplu. Singura diferenta este ca vei avea nevoie de mai multi castigati pentru avansa la skill-ul urmator. Spre exemplu, pentru joburile normale, in avansarea de la de la skill-ul Bronze la Silver ai nevoie de 15.000$ castigati, iar in cadrul jobului pasiv vei avea nevoie de 30.000$(x2 se aplica pentru toate skill-urile). Pentru mai multe detalii cu privire la avansarea in skill, click aici -> https://wiki.liberty.mp/jobs/#cum-avansez-in-skill